Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Docker cheatsheet
```

Comment:
I usually wouldn't comment so pedantically, but as a docker noob, I feel this will help me for a while 😛